lxd/db.go

```diff
@@ -12,6 +12,11 @@ import (
 	_ "github.com/mattn/go-sqlite3"
 )
 
+var (
+	DbErrAlreadyDefined = fmt.Errorf("already exists")
+	NoSuchImageError    = errors.New("No such image")
```

Comment:
They both end up doing the same (instanciate an error object).

I moved both at the beginning of the file without changing them, since they were scattered before, but yeah, I'll probably send a PR to rationalize all of them once I finish fishing out all the various vars and refactor the functions in db.go :)